docs/DeveloperGuide.md

```diff
@@ -467,7 +465,7 @@ MSS:
 1. User selects a criteria to sort the list by
 1. The list items are reordered according to the chosen criterion
 
-_{More to be added}_
+*{More to be added}*
```

Comment:
Ditto on coding style violation. Let us keep to `_` for italics and `**` for bolded text,